src/index.html

```diff
@@ -1,12 +1,370 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" >
+    <link rel="stylesheet" href="./styles/main.css" >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <a href="#home" class="header__link">
+          <img
+            class="header__logo"
+            src="./images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="nav">
```

Comment:
See checklist about multiline siblings.
```suggestion
        </a>
        
        <nav class="nav">
```